modules/gdscript/gdscript.cpp

```diff
@@ -379,10 +379,14 @@ void GDScript::_update_exports_values(Map<StringName, Variant> &values, List<Pro
 }
 #endif
 
-bool GDScript::_update_exports() {
+bool GDScript::_update_exports(bool *p_ret_err, bool p_recursive_call) {
 
 #ifdef TOOLS_ENABLED
 
+	static Vector<GDScript *> base_caches;
+	if (!p_recursive_call) base_caches.clear();
```

Comment:
Please avoid putting both the condition and the statement on the same line. (Same below.)

Reply:
done, now I also have to avoid `condition && func();` 😄